pulp_docker/app/urls.py

```diff
@@ -1,9 +1,19 @@
 from django.conf.urls import url
 
 from pulp_docker.app.views import (
-    VersionView
+    VersionView,
+    BlobManifestView,
+    TagView,
+    TagsListView
 )
 
 urlpatterns = [
     url(r'^v2/$', VersionView.as_view()),
+    url(r'^v2/(?P<path>.+)/tags/list$', TagsListView.as_view()),
+    url(r'^v2/(?P<path>.+)/manifests/(?P<digest>sha256:[A-Fa-f0-9]+)$',
+        BlobManifestView.as_view()),
+    url(r'^v2/(?P<path>.+)/manifests/(?P<tag_name>[A-Za-z0-9\._-]+)$',
```

Comment:
Seems like we might need a validation for Tag.name, we could have a collision if the name was "sha256:pleaseno"

Reply:
Let's file a bug about this.

Reply:
https://pulp.plan.io/issues/4190